embedded-compositor/ScreenShotInterface.hpp

```diff
@@ -0,0 +1,45 @@
+/*!
+ * @file   ScreenShotInterface.hpp
+ *
+ * @class  ScreenShotInterface
+ *
+ * @brief  This class takes a screenshot of the current QML view.
+ *         The image is stored locally with the current timestamp in the filename.
+ *         The image format is PNG.
+ *
+ * @date   10.07.24
+ *
+ * @author Christian Happ
+ *
+*/
+
+#pragma once
+
+#include <QObject>
+#include <QQmlApplicationEngine>
+#include <QString>
+#include "dbusinterface.h"
+
+class ScreenShotInterface : public DBusInterface
```

Comment:
I think we have to rename this file to -> `ScreenShotDBusInterface` because it can confuse with `Interface` form C++

Reply:
done, we should also take this into account by splitting up the `dbusinterface.h` :+1: